Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>Is it used as a prior? I think only likelihood for binary events. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>